apps/src/templates/teacherDashboard/LoginTypeCard.jsx

```diff
@@ -0,0 +1,100 @@
+import React, {Component, PropTypes} from 'react';
+import color from '../../util/color';
+import ProgressButton from '../progress/ProgressButton';
+
+const styles = {
+  card: {
+    overflow: 'hidden',
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'space-between',
+    // Set width to form a three-column layout on 970px teacher dashboard.
+    width: 280,
+    // Uniform height, even in different rows
+    height: 200,
+    paddingTop: 15,
+    paddingLeft: 20,
+    paddingRight: 10,
+    paddingBottom: 15,
+    marginBottom: 20,
+    borderStyle: 'solid',
+    borderWidth: 1,
+    borderColor: color.border_gray,
+    background: color.white
+  },
+  rtlCard: {
+    paddingLeft: 10,
+    paddingRight: 20,
+  },
+  title: {
+    paddingTop: 10,
+    paddingBottom: 5,
+    fontSize: 18,
+    fontFamily:'"Gotham 4r", sans-serif',
+    zIndex: 2,
+    color: color.dark_charcoal,
+    fontWeight: 'bold'
+  },
+  subtitle: {
+    fontSize: 14,
+    fontFamily:'"Gotham 4r", sans-serif',
+    zIndex: 2,
+    color: color.dark_charcoal,
+    fontWeight: 'bold'
+  },
+  description: {
+    paddingTop: 10,
+    paddingBottom: 5,
+    fontSize: 14,
+    fontFamily: '"Gotham 4r", sans-serif',
+    zIndex: 2,
+    color: color.dark_charcoal
+  },
+  button: {
+    alignSelf: 'flex-start',
+    marginTop: 20,
+    marginBottom: 10,
+    zIndex: 2,
+  },
+};
+
+class LoginTypeCard extends Component {
+  static propTypes = {
+    title: PropTypes.string.isRequired,
+    subtitle: PropTypes.string,
+    description: PropTypes.string.isRequired,
+    buttonText: PropTypes.string.isRequired,
+    link: PropTypes.string.isRequired,
+    isRtl: PropTypes.bool.isRequired
+  };
+
+  render() {
+    const { title, subtitle, description, buttonText, link, isRtl } = this.props;
+    const cardStyle = isRtl ? {...styles.card, ...styles.rtlCard} : styles.card;
```

Comment:
Could also be expressed as
```
const cardStyle = {
  ...styles.card,
  ...(isRtl && styles.rtlCard)
};
```
Which I think makes it slightly clearer that they share a base style (card) and extend in the case of rtl.